examples/python/readme.md

```diff
@@ -1,11 +1,10 @@
+# Compile to Python
```

Comment:
Github rendered this weird, what happened here is that I removed an empty line before the first heading. This is a test change to see if you @windelbouwman like it, because there're more cases like that. IMHO, it doesn't make sense to have such empty line at the start of doc.

Reply:
Okay, I'm fine with the empty line or not. Does it not collide with the shbang?

Reply:
> Okay, I'm fine with the empty line or not. Does it not collide with the shbang?

Well, this is readme, and I meant documentation files in my original comment. So yep, no collision with shebang ;-). If I spot "stray" empty lines in executable scripts, I'll raise a separate note.